website/blog/2025-05-08-ferretdb-v220-arm-architecture-support.md

```diff
@@ -0,0 +1,65 @@
+---
+slug: ferretdb-v220-arm-architecture-support
+title: 'FerretDB v2.2.0: `arm64` architecture support and more'
```

Comment:
Did you not notice it in the preview? https://b82b6cfb.ferretdb-blog-dev.pages.dev/ferretdb-v220-arm-architecture-support/ Backticks are not supported there